_docs/faq.md

```diff
@@ -1315,3 +1315,52 @@ IMEx will only call an interaction 'direct' when performed with 2 purified molec
 
 For more details, please see the [corresponding entry](http://www.ebi.ac.uk/intact/about/faq?conversationContext=5#8){:target="blank"} on the the IntAct FAQ page.
 
+## What is a GO-CAM?
+
+GO-CAM stands for Gene Ontology Causal Activity Model. GO-CAMs link
+multiple GO annotations into an integrated model of a biological
+system. More information can be found on the [GO-CAM site](http://geneontology.org/cam).
+
+## What exchange format is used for GO-CAMs?
+
+GO-CAMs include more information than classic GO annotations, so
```

Comment:
"_classic_" or "_standard_"?  http://geneontology.org/docs/tools-overview/ says "GO-Causal Activity Models (GO-CAMs) use a defined “grammar” for linking multiple standard GO annotations into larger models of biological function "

I don't have a preference, just think we should be consistent. @thomaspd what's the Official GO Terminology for this?

Reply:
also line 1353 same question

Reply:
We used "standard" in the paper, so it's probably easiest to stick with this.